Expand Works page documentation

ApeKattQuest-MonkeyPython opened the pull request:
Create the pull request for work on work (and finishing up the artist page)

ApeKattQuest-MonkeyPython committed:
Update author.md

ApeKattQuest-MonkeyPython committed:
Update disambiguation.md

ApeKattQuest-MonkeyPython committed:
Update author.md

ApeKattQuest-MonkeyPython committed:
Update author.md

ApeKattQuest-MonkeyPython committed:
Update author.md

ApeKattQuest-MonkeyPython committed:
Update author.md

ApeKattQuest-MonkeyPython committed:
Update author.md

ApeKattQuest-MonkeyPython committed:
Update author.md

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update disambiguation.md

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,87 +1,115 @@
-Answer: NO. However, we should clarify that it is not about length but more about conceptual "content": for example a haiku is very short but would deserve a work.
-Might link to notes about quotations/exceprts as well (wait could this be handled by quotations?! eg "contains quotation by {author}" i.e. a new rel??)
+- How big is a thing before it is a work?<br>
+Like, is a sentence by Famous Author that is put in the pretext of a lot of Books a "work" in and of itself?<br>
```
I haven't seen 'pretext' used in that context, I think it's a false friend: pretext ≈ excuse, reason

```suggestion
For example, should a quote by Famous Author in the front matter of multiple Books be a separate "work"?<br>
```

MonkeyDo committed:
Work page formatting

ApeKattQuest-MonkeyPython committed:
Update author.md

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

MonkeyDo committed:
Add nl2br extension

Removes the need for adding a "<br>" HML tags manually, and considers line breaks as a carriage returns.

MonkeyDo committed:
Reformat author page, add examples

MonkeyDo committed:
Remove unused footnote

MonkeyDo committed:
Add more examples

MonkeyDo committed:
Improve author credits examples

Add actual examples that fit the bill

MonkeyDo committed:
Fix formatting of author credits examples

jwillikers commented on docs/style/entities/author.md:
```diff
@@ -17,29 +17,30 @@ Try to create short, concise disambiguations.<br>
+Use this for groups of Authors, like [collective pseudonyms](https://en.wikipedia.org/wiki/Category:Collective_pseudonyms) (pseudonyms used by multiple individuals), [house pseudonyms](https://en.wikipedia.org/wiki/Pen_name#Collective_names) (owned by a publisher and used by the writers working for them) and similar.<br>
+Examples:<br>
+[Ellery Queen](https://bookbrainz.org/author/a2532a57-11fa-43d4-aa5d-c2892a2d0320), the group pseudonym of *Frederic Dannay* and *Manfred Bennington Lee*.<br>
+[James S. A. Corey](https://bookbrainz.org/author/c5c71242-f54c-4327-9510-a81064a9a852) pen name used by *Daniel Abraham* and *Ty Franck*.
```
I just recently worked with a lettering studio in BB that is a group: [AndWorld Design](https://bookbrainz.org/author/9268a7af-2f90-49d8-91a1-9684620aaae9).

jwillikers commented on docs/style/entities/author.md:
```diff
@@ -60,7 +61,7 @@ should be related to
 
 When entering books into BookBrainz, it's important to understand the difference between the concept of an <u>Edition</u>, the physical item you can hold in your hands, and the literary concept of <u>Works</u> that books contain. (see also [Wikipedia](https://en.wikipedia.org/wiki/Creative_work) for more information.)<br>
-{add an example of small collection of short stories by differnt author but only one autor name on the cover}
+{add an example of small collection of short stories by different author but only one autor name on the cover}
```
```suggestion
{add an example of small collection of short stories by different author but only one author name on the cover}
```

ApeKattQuest-MonkeyPython committed:
Update docs/style/entities/author.md

Co-authored-by: Nicolás Tamargo <reosarevok@metabrainz.org>

ApeKattQuest-MonkeyPython committed:
Update docs/style/entities/author.md

Co-authored-by: Nicolás Tamargo <reosarevok@metabrainz.org>

ApeKattQuest-MonkeyPython committed:
Update docs/style/entities/author.md

Co-authored-by: Nicolás Tamargo <reosarevok@metabrainz.org>

ApeKattQuest-MonkeyPython committed:
Update docs/style/entities/author.md

Co-authored-by: Nicolás Tamargo <reosarevok@metabrainz.org>

ApeKattQuest-MonkeyPython committed:
Update docs/style/entities/author.md

Co-authored-by: Nicolás Tamargo <reosarevok@metabrainz.org>

ApeKattQuest-MonkeyPython committed:
Update docs/style/entities/author.md

Co-authored-by: Nicolás Tamargo <reosarevok@metabrainz.org>

ApeKattQuest-MonkeyPython committed:
Update docs/style/entities/author.md

Co-authored-by: Jordan Williams <jordan@jwillikers.com>

MonkeyDo committed:
Apply suggestions from code review

Co-authored-by: Nicolás Tamargo <reosarevok@metabrainz.org>

MonkeyDo committed:
Use publisher house name example

Carolyn Keene from the Nancy Drew series

MonkeyDo committed:
Add example for Edition/Work difference

MonkeyDo committed:
Misc wording and grammatical fixes

MonkeyDo committed:
Merge branch 'add-toc' into fixup-author-page

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

jwillikers commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,167 @@
-- [Republic](https://bookbrainz.org/work/c1b11cb0-38b0-49ea-88e8-83566c5589ad) was
-translated by [Benjamin Jowett](https://bookbrainz.org/author/dc98466f-9e81-4fb1-a714-3b62b625b455)
+- a relationship added to the original work it is a translation of
+- a relationship to the author that wrote the original (provided story)
```
It will be nice to have this match what's in the author page regarding using the "provided story" relationship with the original writer.

ApeKattQuest-MonkeyPython committed:
Update work.md

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,176 @@
-or reconstructed. Content is not considered a distinct work if it only
-contains minor variations such as spelling or typographical differences.
+A work is different from another work if there are distinct differences to it, like adaptations, translations, rewritings, and so on.<br>
+A work is not a separate one if there are only minor spelling corrections or typographical changes (these aren't on the level of conceptional changes, only edit(orial) ones).
```
```suggestion
A work is not considered separate if there are only minor spelling corrections or typographical changes (these aren't on the level of conceptional changes, only editorial ones).
```

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,176 @@
-distinct works. Poems, essays, articles and short stories *are* generally
-considered distinct works.
+Usually divisions like chapters are considered integral to the work and not distinct from it. While poems, essays, articles and short stories *are* considered distinct.<br>
+Examples:
```
Missing some examples here

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,176 @@
-it. In general, chapters of a novel are *not* considered to be their own
-distinct works. Poems, essays, articles and short stories *are* generally
-considered distinct works.
+Usually divisions like chapters are considered integral to the work and not distinct from it. While poems, essays, articles and short stories *are* considered distinct.<br>
```
We could add a sub-section for this with a title like:
`#### What about a book's introduction/foreword/preface?`
That way we can link to it (for example underneath in the  "When to create new works?" section)

Discussed on IRC about introduction/forewords/etc. and came up with this:

"If you want to register that an author wrote an introduction or foreword to a novel, for example, you can simply use the "Author wrote introduction for Edition" relationship.
Since more details cannot be captured that way, if needed you can create a separate Work entity to capture those details.
For example: the introduction is denoted with emphasis (for example "Foreword by XXXXX" on the cover), contains more literary content or is a whole essay in itself, or otherwise has notable literary or historical value."

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,176 @@
-Like, is a sentence by Famous Author that is put in the pretext of a lot of Books a "work" in and of itsel?
-Answer: NO. However, we should clarify that it is not about length but more about conceptual "content": for example a haiku is very short but would deserve a work.
-Might link to notes about quotations/exceprts as well (wait could this be handled by quotations?! eg "contains quotation by {author}" i.e. a new rel??)
+What is the difference between an anthology, a short story cycle and a book of short stories?
```
Was this supposed to be a separate section?

```suggestion
### What is the difference between an anthology, a short story cycle and a book of short stories?
```

MonkeyDo replied:
I don't know that I have a clear definition for the difference between those definitions...

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,176 @@
+    * translations
+        * Example:
+
+- Not to create works for simple "introduction/preface/foreword" but to create a link between the Edition and Author instead UNLESS the specific part has become a Thing unto it self and is re-used (cf the discussion in the community thread)
```
See note left above regarding introduction/forewords. Could simply point at the sub-section above.

ApeKattQuest-MonkeyPython committed:
Update work.md

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,178 @@
+3. It is a translation:
+        * Example: [Alice i eventyrland](https://bookbrainz.org/work/50437b54-68db-4c26-871a-23d1546758d7) is a translation of [Alice's Adventures in Wonderland](https://bookbrainz.org/work/337a8c0a-3574-404e-abb5-df606cc36109)
+
+* Usually we do not create works for simple introductions, prefaces and forewords.<br>
```
```suggestion
        * See [What about translations?](#what-about-translations) section
```

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,178 @@
-- [Republic](https://bookbrainz.org/work/c1b11cb0-38b0-49ea-88e8-83566c5589ad) was
-translated by [Benjamin Jowett](https://bookbrainz.org/author/dc98466f-9e81-4fb1-a714-3b62b625b455)
+- a relationship added to the original work it is a translation of
+- a relationship to the author that wrote the original (provided story)
```
```suggestion
- a relationship to the author that wrote the original (see [provided story](./author#current-author-linking-relationships-are) relationship)
```

ApeKattQuest-MonkeyPython committed:
Update work.md

ApeKattQuest-MonkeyPython committed:
Update work.md

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,180 @@
+Examples:<br>
+
+* [VALIS](https://bookbrainz.org/work/983a2d75-522a-41fc-9657-0c6fcf8964c7) - Philip K Dick
+* [<script>alert("!Mediengruppe Bitnik");</script>](https://bookbrainz.org/work/160fc57a-87f5-4adc-ba19-0384fa382cf8) 
```
This is actually triggering a javascript alert on the page.
This formatting avoids running the code:
```suggestion
* [`<script>alert("!Mediengruppe Bitnik");</script>`](https://bookbrainz.org/work/160fc57a-87f5-4adc-ba19-0384fa382cf8) 
```

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
+a short story cycle:
+example:
+book of short stories
+example:
```
Commenting this out until we know what question we want to ask and what the answer actually is (see discussions in https://community.metabrainz.org/t/what-should-i-do-with-the-works-on-this-edition/767562):

```suggestion
<!-- ### What is the difference between an anthology, a short story cycle and a book of short stories?

An anthology is:
example:
a short story cycle:
example:
book of short stories
example: -->
```

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
+
+#### Writer
+The main Author-Work relationship is that of a work to it's author(s), aka its *writer(s)*; most every work should have at least one of these.<br>
+See the [Author](./author.md) page for a description of the types of authors and also the difference between the "written by" Work relationship and Author Credits on Editions.
```
```suggestion
See the [Author](./author.md) page for a description of the types of authors and also the difference between the "written by" Work relationship and [Author Credits](./author.md#author-credits) on Editions.
```

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
+See the [Author](./author.md) page for a description of the types of authors and also the difference between the "written by" Work relationship and Author Credits on Editions.
+
+#### Others
+Notably there is one Work-Author link we'll cover here; That of a work that is *about* an Author; Biographies or bio's.<br>
```
```suggestion
Notably there is one Work-Author relationship we'll cover here: that of biographies, a work that is *about* an Author entity.<br>
```

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
+See [series page](./series.md)
+
+
+[^1]: the graveyard book is  great counter example here because one chapter was released as a short-story prior, another is the short story in "an apple for the creature" which would go on to be basically most of the first chapter of "The girl with all the gifts"
```
Whether this is added as an example above or another example is chosen, this note will need to be deleted.

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
+   specify "parafrases" -> cites is these are the sources of that text
+   explain the difference between citation and excerpt. excerpt is a section of text verbatim, a paragraph taken from whatever
+a citation is instead a work (book) used as a source for whatever in *this* work, usually this is more common for non-fiction.
+reference material will often have a bunch of these.
```
I would remove this text. I'm not sure I understand what it says, but it seems out of place in this list anyway.

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
-### Link to other works
-A work should have a relationship added to another work if:
 
 - *it is derived from another work*, for example: a translation, an adaptation, a revision, a reconstruction, a parody, an inspiration;
```
```suggestion
- derived from another work, for example: a translation, an adaptation, a revision, a reconstruction, a parody, an inspiration;
```

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
-- *it contains the content from another work or is contained in another work*,
-for example: a [fix-up](https://en.wikipedia.org/wiki/Fix-up), a poetry cycle,
-an excerpt, a quote;
+- *it contains the content from another work or is contained in another work*, for example: a [fix-up](https://en.wikipedia.org/wiki/Fix-up), a poetry cycle, an excerpt, a quote;
```
```suggestion
- contains/is contained in another work, for example: a [fix-up](https://en.wikipedia.org/wiki/Fix-up), a poetry cycle, an excerpt, a quote;
```

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
-for example: a [fix-up](https://en.wikipedia.org/wiki/Fix-up), a poetry cycle,
-an excerpt, a quote;
+- *it contains the content from another work or is contained in another work*, for example: a [fix-up](https://en.wikipedia.org/wiki/Fix-up), a poetry cycle, an excerpt, a quote;
 -  for example: a citation.
```
Citation already referenced above, can be removed here

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
-- section above, I'd move these here maybe?
-- (it makes sense that explicit work > author ordered relationship would be here.)
+* contains
+the most important rel, specify that works can belong to several editions, and that an edition can have several works
```
```suggestion
the most important relationship, specify that works can belong to several editions, and that an edition can have several works
```

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
-- (it makes sense that explicit work > author ordered relationship would be here.)
+* contains
+the most important rel, specify that works can belong to several editions, and that an edition can have several works
+editions are the physical book the conceptional work exists in. An edition can contain more than one work, and a work can be published in several editions.
```
```suggestion
Editions are the physical book that the conceptual work exists in. An edition can contain more than one work, and a work can be published in several editions. See [Edition](./edition.md)
```

ApeKattQuest-MonkeyPython committed:
Update docs/style/entities/work.md

Co-authored-by: Monkey Do <MonkeyDo@users.noreply.github.com>

MonkeyDo commented on docs/style/entities/work.md:
```diff
@@ -1,126 +1,182 @@
+#### Others
+Notably there is one Work-Author link we'll cover here; That of a work that is *about* an Author; Biographies or bio's.<br>
+This is a work that is _about_ an Author, someone already in the database. However, someone who never wrote anything, but is in some other way notable, for example; if they link other authors together, via parental or marriage relationships, if they are teachers of other authors, if they have several biographies written about them, they may be be added to BookBrainz, despite not having written an ...
+Examples:
```
```suggestion
Example: [Henry Kissinger](https://bookbrainz.org/author/8eca361c-99f3-46c6-9c8a-a53e92d10694) is the subject of multiple biographies but has not written any work himself
```

MonkeyDo committed:
Commit changes from PR review

MonkeyDo committed:
Add some examples

MonkeyDo committed:
Update docs/style/entities/work.md

MonkeyDo committed:
Update docs/style/entities/work.md

ApeKattQuest-MonkeyPython reviewed (commented):
only one comment: for the 33't line
"composed of multiple stand-alone chapters" I'd change to "composed of multiple chapters"

because the other chapter's aren't really stand-alone (they're _episodic_ but not really their own thing)

ApeKattQuest-MonkeyPython committed:
Update edition.md

ApeKattQuest-MonkeyPython committed:
a proto norwegian guideline page

I'll work on this later

ApeKattQuest-MonkeyPython committed:
Update edition.md

ApeKattQuest-MonkeyPython committed:
Update norsk-guidelines.md

ApeKattQuest-MonkeyPython committed:
Update norsk-guidelines.md

ApeKattQuest-MonkeyPython committed:
Update edition.md

ApeKattQuest-MonkeyPython committed:
Update edition.md

ApeKattQuest-MonkeyPython committed:
Update norsk-guidelines.md

ApeKattQuest-MonkeyPython committed:
Update norsk-guidelines.md

MonkeyDo committed:
Small adjustments

MonkeyDo committed:
Merge pull request #27 from metabrainz/fixup-author-page

Reformat Author page and add examples

MonkeyDo committed:
Merge branch 'master' into add-toc

MonkeyDo committed:
Cleanup intro/foreword section

MonkeyDo committed:
Merge branch 'add-toc' of github.com:metabrainz/bookbrainz-user-guide into add-toc

MonkeyDo committed:
Add one more author group example
